### DISCUSSION TITLE
MadMax using 64GB RamDisk & M2 Pro?

### DISCUSSION THREAD

**Discussion by nonogrub** [madMAx43v3r/chia-plotter#222] [Q&A] [answered]
is this possible?...The idea is to use my limited resources to the limit because I don't have money to update my computer up to 128GB. What will happen when the RamDisk with 48GB is full? Mad Max continue in the M2?

**Comment by SebMoore** [comment, marked as answer]
No. If the ramdisk is not larger than 105gb, the program will crash. If you do not have 105 or more GB of ram, you cannot use a ramdisk and must instead put everything on the SSD. Unfortunate, but that's just how it goes.

**Reply by nonogrub** [reply]
Thanks a lot for your excellent explanation. Appreciate so much

**Reply by SebMoore** [reply]
No worries!
